File: internal/model/season.go
package model

import (
	"time"
)

type Season struct {
	BaseModel
	Year      int        `gorm:"uniqueIndex;not null" json:"year"`             // 2026
	Name      string     `gorm:"size:64;not null" json:"name"`                 // 2026赛季
	Status    string     `gorm:"size:16;default:'active';index" json:"status"` // active / archived
	StartDate *time.Time `json:"start_date"`
	EndDate   *time.Time `json:"end_date"`
}

func (Season) TableName() string { return "seasons" }

File: frontend-mp/request/season.js
// 赛季模块（公开只读）
const { get } = require('./http');

/**
 * 当前激活赛季：返回 { id, year, name, status, start_date, end_date }
 */
function active() {
  return get('/seasons/active');
}

module.exports = { active };
